src/pages/SubmissionConfig.js

```diff
@@ -163,14 +167,12 @@ export const SubmissionConfig = ({ methodId }) => {
   }
 
   const renderInputs = () => {
-    return configuredInputDefinition ? h(ReactJson, {
-      style: { whiteSpace: 'pre-wrap' },
-      name: false,
-      collapsed: 4,
-      enableClipboard: false,
-      displayDataTypes: false,
-      displayObjectSize: false,
-      src: configuredInputDefinition
+    return configuredInputDefinition ? h(renderInputTable, {
```

Comment:
Incidentally, another great reason to make this a separate component. We almost certainly want to be able to show historical input and output definitions on the Submission Details page (the spiritual successor to the "Show Inputs" button, albeit at a submission level rather than in-table-per-workflow level). 

![image](https://user-images.githubusercontent.com/13006282/208467772-dc6d5c71-f3b7-47eb-8625-9f827e185cc3.png)

Having this be a component that can either be read/write for config or read-only for record keeping would be super nice.